hyperparameter_tuning/blazingtext_text_classification_dbpedia/hpo_blazingtext_text_classification_dbpedia.ipynb

```diff
@@ -0,0 +1,710 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    # Since preprocessing the whole dataset might take a couple of mintutes,
</pre><p>Typo, change <code>mintutes</code> to <code>minutes</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>